tests/test_interface.py

```diff
@@ -43,12 +60,40 @@ def test_InterfaceAddIpv4Address(self, dvs):
 
         assert subnet_found and ip2me_found
 
+    def test_InterfaceChangeMtu(self, dvs):
+        pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+        tbl = swsscommon.ProducerStateTable(pdb, "PORT_TABLE")
+        fvs = swsscommon.FieldValuePairs([("mtu", "8888")])
+        tbl.set("Ethernet8", fvs)
```

Comment:
do we need to configure ethernet8 into rif or it is already in rif.

Reply:
in the first test case when the ip is assigned, the associated rif is created